### ISSUE TITLE
In Android studio 2.3.3 there are no attributes pane

### ISSUE THREAD

**Issue by exalkas** [googlecodelabs/constraint-layout#29]
It's mentioned at #7 as "attributes" but in studio is called "properties" pane

**Comment by kyberkiller** [comment]
Yes, nice trap for the beginners. Very first things totally wrong. No one seems to test this level documentation.
